src/components/tricots/SingleTrikotActual.js

```diff
@@ -28,18 +30,50 @@ class SingleTrikotActual extends Component {
             rankOfBestSwiss = sortedConnections.findIndex(con => con.rider.id === bestSwiss.id) + 1;
         }
 
+      switch(trikot.type) {
+        case 'leader':
+          if (leader !== undefined) {
+            flag = countries.find((v) => {
+              return v.ioc === leader.country;
+            });
+          }
+          break;
+        case 'mountain':
+          if (mountain !== undefined) {
+            flag = countries.find((v) => {
+              return v.ioc === mountain.country;
+            });
+          }
+          break;
+        case 'points':
+          if (point !== undefined) {
+            flag = countries.find((v) => {
+              return v.ioc === point.country;
+            });
+          }
+          break;
+        case 'bestSwiss':
+          if (bestSwiss !== undefined) {
+            flag = countries.find((v) => {
+              return v.ioc === bestSwiss.country;
+            });
+          }
+          break;
+        default:
+          flag = undefined;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Review this useless assignment: "flag" already holds the assigned value along all execution paths. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS4165)